### PR TITLE
Aryanmajeed/issue76

### LINKED ISSUE #76: The Notification Pop-Up
**Is your feature request related to a problem? Please describe.**
The notification pop-up should notify the user if they got any interaction on our platform

**Describe how the solution should behave**
Each interaction causes a notification.

**Screenshots of the design/if any**
notification

### PR DESCRIPTION
**Pull request by aryankarim**
## What?
Adding the notification functionality for likes and comments
## Why?
to notify the user when someone likes or comments on his/her post
## How to use?
Go to the homepage and like or comment on a post that's posted by your account or someone else's to see the notifications
## Screenshots
screencapture-localhost-3000-posts-UDEYX4qI3nxOCPNIchTF-2021-11-03-05_10_11

### CODE REVIEW

File: components/Notifications/NotificationsList.js [hunk 1]
```diff
@@ -5,89 +5,63 @@ import {
...
+import { setNotifications } from "../../store/user/userSlice";
+import { onValue, ref } from "@firebase/database";
+import { GiDesert } from "react-icons/gi";
+
 function NotificationsList() {
+  const dispatch = useDispatch();
+  const { data, status } = useSelector((state) => state.user.notifications);
+
+  useEffect(() => {
+    //note: these listeners should be detached?!
```
Comment (MuhannedNoman) [inline components/Notifications/NotificationsList.js]: Then shouldn't you detach them in the return?

File: pages/posts/[postId].js [hunk 2]
```diff
@@ -55,13 +59,13 @@ export default function Index({ post, user, some }) {
...
-    },
-    resetCommentStatus
-  );
+  // useToastHook(
+  //   {
+  //     status: postCommentStatus,
+  //     error: "Something went wrong",
+  //   },
+  //   resetCommentStatus
+  // );
```
Comment (MuhannedNoman) [inline pages/posts/[postId].js]: Can we remove it if this is not used?

Comment (aryankarim) [reply pages/posts/[postId].js]: yes sure!

**Review by EsraaY21 — APPROVED** [review]
Great work!

File: components/Notifications/NotificationsList.js (hunk 1 above)
Comment (aryankarim) [reply components/Notifications/NotificationsList.js]: Done!

**Review by NooraYaqub44 — APPROVED** [review]
LGTM great job

**Pull request merged**